services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events.py

```diff
@@ -443,14 +442,13 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
             or scheduler_data.dynamic_sidecar.swarm_network_name is None
             or scheduler_data.proxy_admin_api_port is None
         ):
-            raise ValueError(
-                "Did not expect None for any of the following: "
-                f"{scheduler_data.dynamic_sidecar.dynamic_sidecar_id=} "
-                f"{scheduler_data.dynamic_sidecar.dynamic_sidecar_network_id=} "
-                f"{scheduler_data.dynamic_sidecar.swarm_network_id=} "
-                f"{scheduler_data.dynamic_sidecar.swarm_network_name=} "
-                f"{scheduler_data.proxy_admin_api_port=}"
+            msg = (
+                f"Did not expect None for any of the following:"
+                f" scheduler_data.dynamic_sidecar.dynamic_sidecar_id={scheduler_data.dynamic_sidecar.dynamic_sidecar_id!r} "
```

Comment:
well to be honnest this was an autofix from Ruff.
but if you check the docs on f-string, !r uses the *repr* of the value given. it would actually be more explicit to just write *repr(value)* instead of *!r*.

Reply:
changed back to original more explicit and concise version